src/languageservice/parser/ast-converter.ts

```diff
@@ -1,4 +1,19 @@
-import { Node, isScalar, Scalar, isMap, YAMLMap, isPair, Pair, isSeq, YAMLSeq, isNode, isAlias, Alias, Document } from 'yaml';
+import {
+  Node,
+  isScalar,
+  Scalar,
+  isMap,
+  YAMLMap,
+  isPair,
+  Pair,
+  isSeq,
+  YAMLSeq,
+  isNode,
+  isAlias,
+  Alias,
+  Document,
+  LineCounter,
```

Comment:
If you are going to the trouble of putting them each on a line, it might be worth alphabetizing it. 
It's up to you.

Reply:
That is `prettier` does, as line with import becomes to long